scripts/install_avalanchego_release.sh

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env bash
+set -e
+
+# Load the versions
+TELEPORTER_PATH=$(
+  cd "$(dirname "${BASH_SOURCE[0]}")"
+  cd .. && pwd
+)
+source "$TELEPORTER_PATH"/scripts/versions.sh
+
+# Load the constants
+source "$TELEPORTER_PATH"/scripts/constants.sh
+
+############################
+# download avalanchego
+# https://github.com/ava-labs/avalanchego/releases
+GOARCH=$(go env GOARCH)
+GOOS=$(go env GOOS)
+BASEDIR=${BASEDIR:-"/tmp/avalanchego-release"}
+AVALANCHEGO_BUILD_PATH=${AVALANCHEGO_BUILD_PATH:-${BASEDIR}/avalanchego}
+
+mkdir -p ${BASEDIR}
+
+AVAGO_DOWNLOAD_URL=https://github.com/ava-labs/avalanchego/releases/download/${AVALANCHEGO_VERSION}/avalanchego-linux-${GOARCH}-${AVALANCHEGO_VERSION}.tar.gz
+AVAGO_DOWNLOAD_PATH=${BASEDIR}/avalanchego-linux-${GOARCH}-${AVALANCHEGO_VERSION}.tar.gz
+
+if [[ ${GOOS} == "darwin" ]]; then
+  AVAGO_DOWNLOAD_URL=https://github.com/ava-labs/avalanchego/releases/download/${AVALANCHEGO_VERSION}/avalanchego-macos-${AVALANCHEGO_VERSION}.zip
+  AVAGO_DOWNLOAD_PATH=${BASEDIR}/avalanchego-macos-${AVALANCHEGO_VERSION}.zip
+fi
+
+BUILD_DIR=${AVALANCHEGO_BUILD_PATH}-${AVALANCHEGO_VERSION}
+
+extract_archive() {
```

Comment:
A similar function exists in `install_subnetevm_release.sh`. Can we generalize it and move it to `constants.sh` (or some other common script)?